packages/wrangler/src/cli-hotkeys.ts

```diff
@@ -8,6 +8,7 @@ import type { Hook } from "./api";
 export default function (
 	options: Array<{
 		keys: string[];
+		disabled?: boolean;
```

Comment:
Why do we have a `disabled` option? Can't we just not pass it in the array?

Reply:
Felt it was easier to read a disabled flag in an flat-ordered list than the ternary+spread alternative

https://github.com/cloudflare/workers-sdk/pull/6149/commits/9b7e52fe8d1d17f51920a4eb8c1c2a33175c99e0